### PR TITLE
thats a good days work

### PR DESCRIPTION
**Pull request by DiggDiggs**
#3

### CODE REVIEW

File: test_get_count.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python3
```
Comment (Shadi-Shwiyat) [inline test_get_count.py:1]: is this file needed?

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -74,3 +74,33 @@ def reload(self):
...
+    def count(self, cls=None):
+        """
+        count the number of objects in storage
+        """
+        all_class = classes.values()
+
+        if not cls:
+            count = 0
+            for clas in all_class:
+                count += len(models.storage.all(clas).values())
```
Comment (Shadi-Shwiyat) [inline models/engine/db_storage.py:102]: did you mean to have `count += len(models.storage.all(cls).values())` instead of `count += len(models.storage.all(clas).values())`

**Review by Shadi-Shwiyat — APPROVED** [review]
code seems good thanks!

File: test_get_count.py (hunk 1 above)
Comment (DiggDiggs) [reply test_get_count.py:1]: yes it plays in the roll of testing file ensures that the console.py module and the tests/test_console.py module have appropriate docstrings and adhere to the PEP 8 style guidelines.

File: models/engine/db_storage.py (hunk 2 above)
Comment (DiggDiggs) [reply models/engine/db_storage.py:102]: oops 
ya
